resources/assets/js/app.js

```diff
@@ -1293,42 +1294,6 @@ function initAutocomplete() {
     tokenFieldCheck();
   });
 
-  function deviceFormCollect($form) {
```

Comment:
Some of these changes seem related to the work in the unpowered branch, and not just the event stats, so I'm hoping this will all merge in OK?